backend/game/cards/discard_cards/victory_point_manipulation_cards/test_drop_from_high_altitude.py

```diff
@@ -0,0 +1,25 @@
+from game.cards.discard_cards.victory_point_manipulation_cards.drop_from_high_altitude import DropFromHighAltitude
+from game.values.locations import Locations
+
+
+def test_drop_from_high_altitude_adds_2_victory_points(player):
+    DropFromHighAltitude().immediate_effect(player, None)
+    assert player.victory_points == 2
+
+
+def test_drop_from_high_altitude_costs_5_energy():
+    assert DropFromHighAltitude().cost == 5
+
+
+def test_drop_from_high_altitude_location_tokyo():
+    assert DropFromHighAltitude().location == Locations.OUTSIDE
+
+
+def test_drop_from_high_altitude_location_outside_tokyo(five_players):
```

Comment:
This test is super complicated. 
1. Setup
Set player 2 inside Tokyo
2. Action
Player 1 buys card
3. Assert
Assert player 2 is outside and player 1 is inside.

Reply:
true i tweaked the test slightly simplified it in recent commit, it places the first other user in Tokyp, fires off the method and then checks that no other users are in Tokyo as they should all have been moved out